msb/zmq_base/Subscriber.py

```diff
@@ -1,32 +1,38 @@
 import zmq
 import pickle
-import sys
-import time
+import json
 
 
 class Subscriber:
-    def __init__(self, topic = "", protocol="tcp", ip="localhost", port="5555", connect_to = None):
-        self.topic = topic.encode() if type(topic) != bytes else topic
+    def __init__(self, topic_or_topics, config):
```

Comment:
same as with the publisher - i'd prefer it , if the subscriber handles the creation of the configuration autonomously